src/app/helpers/$.js

```diff
@@ -1,10 +1,15 @@
 let $ = {};
 
-$.toggleClass = function toggleClass(el, name) {
-    if (el.classList.contains(name)) {
-        el.classList.remove(name);
+$.setText = (node, value) => {
+    node.innerHTML = '';
+    node.appendChild(document.createTextNode(value));
```

Comment:
Why wouldn't you just do `node.textContent = value` and eschew this function entirely?  This seems to be doing a fair bit more than it implies as well, since it's effectively emptying the node first (`innerHTML = ''` isn't the best way to do that either).